store/tikv/rawkv_test.go

```diff
@@ -16,6 +16,7 @@ package tikv
 import (
 	"bytes"
 
+	"fmt"
```

Comment:
Strange format. `fmt` should be grouped with `bytes`, not the packages below.

Reply:
But this is checked by `make check`

Reply:
I think `make check` is not strict enough.

Reply:
The`make check` will `gofmt` all go file.

Reply:
In fact we always group built-in packages together and other dependencies together, manually.